src/Illuminate/Hashing/BcryptHasher.php

```diff
@@ -67,6 +67,10 @@ public function make(#[\SensitiveParameter] $value, array $options = [])
      */
     public function check(#[\SensitiveParameter] $value, $hashedValue, array $options = [])
     {
```

Comment:
if we don't check for a nullish or empty value first, we'll always throw a runtime exception which means the end user won't receive the normally expected 401 or similar failed to authenticate, but rather a 500 server error which makes applications look broken